src/controllers/validators/StartDateValidator.ts

```diff
@@ -0,0 +1,38 @@
+import { Request } from 'express';
+import { FormValidator } from './FormValidator';
+
+import { schema } from 'app/models/fields/Date.schema';
+import { ValidationError } from 'app/utils/validation/ValidationError';
+import { ValidationResult } from 'app/utils/validation/ValidationResult';
+
+export class StartDateValidator extends FormValidator {
```

Comment:
Same here, class could be named in more generic way to make it reusable for other date screens.